lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -19,6 +19,18 @@ class MiqAeBuiltinMethod
     SERVICE        = 'service'.freeze
     UNKNOWN        = 'unknown'.freeze
 
+    def self.find_miq_request_id(workspace)
```

Comment:
I'm thinking this should be a method on the MiqAeWorkspace itself and that would DRY up the code.  Then below you could call `obj.workspace.find_miq_request_id`.  Similarly you could do that up above in MiqAeEngine.deliver, you could do `ws.find_miq_request_id`.